recurly/response.py

```diff
@@ -12,29 +33,26 @@ def __init__(self, response, request):
         self.status = response.status
         http_body = response.read()
         self.body = None
+        self.__headers = response.headers or {}
+
+        self.request_id = self.__headers.get("X-Request-Id")
+        self.date = self.__headers.get("Date")
+
+        self.rate_limit = parse_int_header(self.__headers, "X-RateLimit-Limit")
+        self.rate_limit_remaining = parse_int_header(
+            self.__headers, "X-RateLimit-Remaining"
+        )
+        self.rate_limit_reset = parse_datetime_header(
+            self.__headers, "X-RateLimit-Reset"
+        )
+        self.content_type = self.__headers.get("Content-Type", "").split(";")[0]
+
+        self.proxy_metadata = {
+            "server": self.__headers.get("Server"),
+            "cf-ray": self.__headers.get("CF-RAY"),
+        }
+
+        self.total_records = parse_int_header(self.__headers, "recurly-total-records")
```

Comment:
@douglasmiller why is this lowercase when every other header is upper?

Reply:
Maybe `headers` is some kind of case insensitive dict

Reply:
Yes, that's actually supposed to be upper, but yes it's probably case insensitive. I would support making it correct `Recurly-Total-Records`